CODIGO/General.bas

```diff
@@ -1133,25 +1133,22 @@ On Error GoTo Main_Err
     CheckMD5 = GetMd5
     SessionOpened = False
     
-
-    Call Load(frmConnect)
-    Call Load(FrmLogear)
-        
+    If CheckAndSetBabelUIUsage Then
+        Call InitializeUI(D3DWindow.BackBufferWidth, D3DWindow.BackBufferHeight, 4)
```

Comment:
Also we should print a message `debug.print Babel Failed` if the initialization fails for any reason

Reply:
adding log error inside Public Sub InitializeUI(ByVal Width As Long, ByVal Height As Long, ByVal pixelSize As Long)